### ISSUE TITLE
font-icon has a offset inside icon-button

### ISSUE THREAD

**Issue by cebor** [angular/material#3149]
Im using v0.9.8-rc1:

The new font icons have minimal offset to the bottom when being used inside a icon-button!

SVGs are fine.

```html
<md-button class="md-icon-button" aria-label="Face">
  <md-icon>face</md-icon>
</md-button>
```

**Comment by vanntian** [comment]
+1 i have the same problem

**Comment by jlmagee** [comment]
There is certainly some mystery with the alignment. This plnkr has an example of a pretty well aligned icon but it falls apart when an href is included (changes button element to anchor element).

http://plnkr.co/edit/mHC8srfdVs3OoSOeolX8

**Comment by cebor** [comment]
same issue on v0.9.7 with svg icons

**Comment by jlmagee** [comment]
Appears to be fixed in  0.10-rc1 

button: fixes style inconsistencies between md-buttons using a and button under th (cc07e63d, closes #2440)
